app/src/main/java/app/revanced/integrations/youtube/patches/components/DescriptionComponentsFilter.java

```diff
@@ -45,20 +44,42 @@ public DescriptionComponentsFilter() {
                 "transcript_section"
         );
 
+        macroMarkersCarousel = new StringFilterGroup(
+                null,
```

Comment:
Why is this null and the two groups below not?

Reply:
`macro_markers_carousel.eml` is the common string which hides both chapters and key concepts having bytearray `chapters_horizontal_shelf` and `learning_concept_macro_markers_carousel_shelf`. So if I use a setting value for macroMarkerCarousel, wouldn't it restrict to either one of the bytearray?

I'm not sure myself, feel free to suggest any required changes as you feel.